apps/hash-api/src/graphql/resolvers/ontology/entity-type.ts

```diff
@@ -136,7 +135,7 @@ export const getEntityTypeResolver: ResolverFn<
     },
   });
 
-  return mapSubgraph(entityTypeSubgraph);
+  return entityTypeSubgraph as Subgraph;
```

Comment:
```suggestion
  return entityTypeSubgraph as Subgraph< EntityTypeRootType>;
```

Reply:
Applied in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)